docs/en/reference/transactions.rst

```diff
@@ -258,4 +258,3 @@ If you need stricter control, you can catch the concrete exceptions directly:
 - ``Doctrine\DBAL\Exception\LockWaitTimeoutException``: this exception happens when
   a transaction has to wait a considerable amount of time to obtain a lock, even if
   a deadlock is not involved.
-
```

Comment:
Please don't remove the last new line at the end of the file.

Reply:
Fixed.